src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -0,0 +1,226 @@
+package com.wikia.webdriver.testcases.portableinfoboxtests;
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.wikitextshortcuts.WikiTextShortCutsComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.PortableInfoboxPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.editmode.SourceEditModePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.category.CategoryPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialWhatLinksHerePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.themedesigner.SpecialThemeDesignerPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.template.TemplatePageObject;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+/**
+ * Created by Rodriuki on 12/06/15.
+ * Set of Test Cases found on https://one.wikia-inc.com/wiki/Portable_Infoboxes_Test_Plan
+ *
+ * TC01: Verify elements visibility: infobox title, image, headers, italic, bold, quotation marks, references
+ * TC02: Verify correct redirects in mediawiki119.wikia.com/wiki/RodriInfobox01 for:
+ * external links, internal links, red links
+ * TC03: Verify images used in infoboxes appear in Special:WhatLinksHere page
+ * TC04: Verify adding a category to infobox markup and then invoking that template in an article
+ * page will display category in categories section at the bottom of the page automatically
+ *
+ * Created by nikodamn 20/07/15
+ * TC05: Verify lightbox opens when clicking infobox image
+ * TC06: Verify visibility of tabber and it's images
+ * TC07: Verify infobox color has changed after changing colors in wiki Theme Designer
+ * TC08: Verify if ordered and unordered lists are parsed correctly after adding them
+ * TC13: Verify category links inside infoboxes
+ * TC14: Verify if horizontal group font size matches other elements font
+ * TC15: Copy syntax from template page to article and verify presence of all new information provided
+ */
+public class PortableInfoboxTests extends NewTestTemplate {
+
+  private final static Logger LOGGER = Logger.getLogger(PortableInfoboxTests.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)